flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -45,7 +45,7 @@ function wait_elasticsearch_working {
     echo "Waiting for Elasticsearch node to work..."
 
     for ((i=1;i<=60;i++)); do
-        curl -XGET 'http://localhost:9200'
+        curl -XGET 'http://localhost:9200' || true
```

Comment:
how can changes to the kafka tests break this one?

Reply:
`test_sql_client.sh` tests the Kafka and ElasticSearch connector in one test.

Reply:
ah. so the `test_sql_client` includes `kafka-common` which uses `set -e`, which then applies to everything the test executes?

Reply:
Exactly. That's why setting `set -e` in some common script can be a pain in the ass.

Reply:
Agreed. Now all changes make sense, +1.